runtime/doc/treesitter.txt

```diff
@@ -461,7 +467,7 @@ parse_query({lang}, {query})                                   *parse_query()*
                 can be used to search nodes in the syntax tree for the
                 patterns defined in {query} using `iter_*` methods below.
 
-                Exposes `info` and `captures` with additional information about the {query}.
+                Exposes `info` & `captures` with additional info about {query}.
```

Comment:
I've reworded a few things to make them fit but some changes don't seem ideal.

Here `info`, the table clashes with "info", the word